### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [janetvn/coderschool_mc_w06_unsupervised_learning_assignment_Nguyen_Ngoc_Mai#1]
---
# Assignment 6

# Goal of this Assignment

The goal of this assignment was to introduce you to following concepts:
* Unsupervised Learning (KMeans, Hierachical Clustering)
* PCA

You learn how to use PCA for dimension reduction, KMeans, and Hierarchical Clustering. Also you learn to visualize the result of both tenichque.

# Things you did well:

* Ultilizing Pandas to read, summary and visualize data.
* Got everything right

# To sum up:

* Great work! This showing you understood the concept and able to apply it. Keep it up!
